src/QuickMarcEditor/QuickMarcEditorRows/FixedField/FixedField.css

```diff
@@ -8,3 +8,16 @@
   width: 70px;
   display: inline-block;
 }
+
+.collapsedField {
+  margin-right: 10px;
+}
+
+[class*="headerWrapper"] [class*="header"] {
```

Comment:
why not to add own class?

Reply:
This styles for accordion header, we have no way to set className directly (styles for Accordion tag isn't work)

Reply:
but you defined global styles